services/course-material/src/components/SearchDialog.tsx

```
@@ -99,6 +99,16 @@ const SearchDialog: React.FC<SearchDialogProps> = ({ courseId, organizationSlug
     setOpen(true)
   }
 
+  useEffect(() => {
```

Don't do this, ignore the error